WORKSPACE

```diff
@@ -73,3 +72,22 @@ http_archive(
 load("@bazel_skylib//:workspace.bzl", "bazel_skylib_workspace")
 
 bazel_skylib_workspace()
+
+# Stardoc (for documentation)
+
+stardoc_version = "0.4.0"
+
+stardoc_sha = "6d07d18c15abb0f6d393adbd6075cd661a2219faab56a9517741f0fc755f6f3c"
+
+http_archive(
+    name = "io_bazel_stardoc",
+    sha256 = stardoc_sha,
+    strip_prefix = "stardoc-{v}".format(v = stardoc_version),
+    urls = [
+        "https://github.com/bazelbuild/stardoc/archive/{v}.tar.gz".format(v = stardoc_version),
```

Comment:
We use `.zip` for other `http_archive`s

Obligatory xkcd about `tar`https://www.xkcd.com/1168/

![](https://imgs.xkcd.com/comics/tar_2x.png)

Reply:
Actually, the only ZIP one is `rules_kotlin`, which I’m changing to tarball. You know, for consistency. Thanks for pointing out! For reals though — almost every rule uses tarballs for official releases (in addition to the default GitHub ZIP / tarball pair).